:book: Remove erroneous ref to CSV output (#1813)
ossf/scorecard: README.md

##### Formatting Results

> There are two formats currently: `default`, and `json`. Others may be
> added in the future.

Reviewer: ```suggestion
The currently supported formats are `default` (table) and `json`.
```

Author: Done.
